### PR TITLE
Update Chart.js to version 2.7.1

### PR DESCRIPTION
**Pull request by cauemello**

### CODE REVIEW

**Comment by cauemello** [comment]
@jackiekazil ,

Can you review this update?

Open issue: #284

File: mesa/visualization/templates/js/TextModule.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
 var TextModule = function() {
-	var tag = "<p></p>";
+	var tag = "<p class='lead'></p>";
```
Comment (TaylorMutch) [inline mesa/visualization/templates/js/TextModule.js]: The class usage here doesn't seem to serve any purpose here., either from a CSS rule or used in the frontend code. Is there a use that you have found for it?

Comment (cauemello) [reply mesa/visualization/templates/js/TextModule.js]: This is a bootstrap class to "Make a paragraph stand out".
https://getbootstrap.com/docs/4.0/content/typography/#lead

Comment (TaylorMutch) [reply mesa/visualization/templates/js/TextModule.js]: Ah okay, cool!

**Comment by TaylorMutch** [comment]
I see no issues with merging this in. Merge away @jackiekazil?

**Comment by TaylorMutch** [comment]
@cauemello Would it be possible for you to do a rebase of this branch on top of origin/master? There are a lot of commits in this PR that are just merges of other PRs - rebasing would help make sure that there's a clearer representation of what this PR contains when it gets merged in.

**Comment by cauemello** [comment]
@TaylorMutch 
Is this ok?
Thank you

**Comment by TaylorMutch** [comment]
@cauemello at a glance this looks much better! I will test this afternoon and then likely merge this in. Thank you for all your work on this!

**Comment by TaylorMutch** [comment]
Closed #401.

**Pull request merged**